native/build.gradle

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'java'
+    id 'com.github.spotbugs'
+    id 'checkstyle'
+    id 'jacoco'
+    id 'maven-publish'
+}
+
+description = 'Ballerina - Persist Java Native'
+
+configurations {
+    jacocoRuntime
+}
+
+dependencies {
+    jacocoRuntime "org.jacoco:org.jacoco.agent:${jacoco.toolVersion}:runtime"
+
+    checkstyle project(":checkstyle")
+    checkstyle "com.puppycrawl.tools:checkstyle:${puppycrawlCheckstyleVersion}"
+    implementation group: 'org.ballerinalang', name: 'ballerina-lang', version: "${ballerinaLangVersion}"
+    implementation group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
+    implementation group: 'io.ballerina.stdlib', name: 'sql-native', version: "${stdlibSqlVersion}"
```

Comment:
```suggestion
```

Reply:
fixed in [caecb93](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/1/commits/caecb93da85c1d1626d09ea912843b8249d12237)